src/test-utils/mock-api-client.ts

```diff
@@ -1,8 +1,9 @@
 import { ApiClient } from '../api';
-import { createFakeReportingConfigResponse } from './fake-reporting-config-reponse';
 
 export function createMockApiClient(): ApiClient {
 	return {
-		loadReportingConfig: jest.fn( async () => createFakeReportingConfigResponse() ),
+		loadReportingConfig: jest.fn( async () => {
```

Comment:
I decided it was overkill to return some hardcoded value here. For now, we'll just have this return an empty reporting config. Tests can easily override this mock function.